### PR TITLE
Add Portuguese translations for some templates

### PR DESCRIPTION
**Pull request by MichelleCordeiro**

### CODE REVIEW

File: templates/pt/rectification-default.txt [hunk 1]
```diff
@@ -0,0 +1,17 @@
+A quem possa interessar:
+
+Solicitando a retificação dos meus dados pessoais incorretos, de acordo com o Artigo 16 do RGPD.
+
+As seguintes alterações devem ser feitas:
+{rectification_data}
+Caso você tenha divulgado os dados pessoais incorretos a terceiros, deverá informar minha solicitação de retificação a todos os destinatários, conforme estabelecido no Artigo 19 do RGPD. Por favor, também me informe sobre esses destinatários.
+
+Minha solicitação inclui explicitamente [runs > the following as well as] e quaisquer outros serviços e empresas para os quais você é o responsável, conforme definido no artigo 4 (7) do GDPR [runs>: {runs_list}].
```
Comment (baltpeter) [inline templates/pt/rectification-default.txt]: This one is not very clear, sorry.

The `the following as well as` needs to be translated, as well.

Then, please have no spaces around the `>`, i.e.: `[runs>your translation here]`.

Comment (MichelleCordeiro) [reply templates/pt/rectification-default.txt]: Sorry, I thought it wasn't necessary.

Comment (baltpeter) [reply templates/pt/rectification-default.txt]: No worries!

**Comment by baltpeter** [comment]
Thanks for the translation!

Not sure if you've seen this already but feel free to signup to our Hacktoberfest event to get some stickers from us, if you want: https://www.datarequests.org/blog/hacktoberfest-2019/

**Comment by marcelosantos89** [comment]
This looks to me pt-br not pt-pt .. do you want pt-pt also ?

**Comment by zner0L** [comment]
@marcelosantos89 Yes, since we target mainly EU citizens, having `pt-pt` would be great. We just decided that we would like the current `pt` directory to be renamed to `pt-br`, so you could add your translations in `pt-pt` to a directory called `pt`.

**Pull request merged**